build.js

```diff
@@ -27,9 +27,9 @@ var environments = require('./environments');
 
 var fs = require('fs');
 var path = require('path');
-var os = require('os');
+// var os = require('os');
```

Comment:
Why are these commented?

Reply:
Because it's required for Closure Compiler tests which now can't be built.